src/core/Akka.Persistence/Journal/AsyncWriteJournal.cs

```diff
@@ -54,9 +56,9 @@ protected AsyncWriteJournal()
             var config = extension.ConfigFor(Self);
             _breaker = new CircuitBreaker(
                 Context.System.Scheduler,
-                config.GetInt("circuit-breaker.max-failures", 0),
-                config.GetTimeSpan("circuit-breaker.call-timeout", null),
-                config.GetTimeSpan("circuit-breaker.reset-timeout", null));
+                config.GetInt("circuit-breaker.max-failures", 10),
+                config.GetTimeSpan("circuit-breaker.call-timeout", TimeSpan.FromSeconds(10)),
+                config.GetTimeSpan("circuit-breaker.reset-timeout", TimeSpan.FromSeconds(30)));
```

Comment:
Seems high IMHO

Reply:
But we can address that in a later issue if it is a problem - in fact I think 30s is the current default, isn't it?